_maps/shuttles/pgf/pgf_elated_bolide.dmm

```diff
@@ -4082,7 +4082,8 @@
 	can_be_unanchored = 1;
 	icon_state = "sec";
 	name = "uniforms locker";
-	req_access_txt = "1"
+	req_access_txt = "1";
+	storage_capacity = 40
```

Comment:
this is a pretty kludgey fix, i'd strongly discourage just varediting for things like this. try to find a way to add a second locker nearby or reduce the amount of items in this locker if possible instead.

Reply:
when you have a large enough hammer (varedits) every problem looks like a nail